### PR TITLE
[Doc] Update Nvidia Jetson TX2 benchmark

### PR DESCRIPTION
**Pull request by SingleZombie**
Note that some models (including mmocr models) are not supported.

### CODE REVIEW

File: docs/zh_cn/benchmark.md [hunk 1]
```diff
@@ -304,98 +333,45 @@ GPU: ncnn, TensorRT, PPLNN
...
 </thead>
 <tbody>
   <tr>
     <td align="center" rowspan="3">Model</td>
     <td align="center" rowspan="3">Input</td>
     <td align="center" colspan="6">T4</td>
+    <td align="center" colspan="2">Jetson TX2</td>
     <td align="center" colspan="2">T4</td>
-    <td rowspan="3">model config file</td>
+    <td align="center" rowspan="3">model config file</td>
```
Comment (AllentDan) [inline docs/zh_cn/benchmark.md:368]: may keep original

File: docs/zh_cn/benchmark.md [hunk 2]
```diff
@@ -304,98 +333,45 @@ GPU: ncnn, TensorRT, PPLNN
...
+
 </div>
 </details>
 
 <details>
 <summary style="margin-left: 25px;">MMEdit</summary>
 <div style="margin-left: 25px;">
-<table class="docutils">
+
+<table class='docutils'>
```
Comment (AllentDan) [inline docs/zh_cn/benchmark.md:352]: may keep original

File: docs/zh_cn/benchmark.md [hunk 3]
```diff
@@ -528,13 +511,14 @@ GPU: ncnn, TensorRT, PPLNN
 <details>
 <summary style="margin-left: 25px;">MMSeg</summary>
 <div style="margin-left: 25px;">
-<table class="docutils">
+
+<table class='docutils'>
```
Comment (AllentDan) [inline docs/zh_cn/benchmark.md:515]: may keep original

File: docs/en/benchmark.md [hunk 4]
```diff
@@ -28,11 +28,11 @@ Users can directly test the speed through [how_to_measure_performance_of_models.
 <summary style="margin-left: 25px;">MMCls</summary>
 <div style="margin-left: 25px;">
 
-<table class="docutils">
+<table class='docutils'>
```
Comment (AllentDan) [inline docs/en/benchmark.md:31]: may keep original

File: docs/en/benchmark.md [hunk 5]
```diff
@@ -175,13 +187,14 @@ Users can directly test the speed through [how_to_measure_performance_of_models.
 <details>
 <summary style="margin-left: 25px;">MMDet</summary>
 <div style="margin-left: 25px;">
-<table class="docutils">
+
+<table class='docutils'>
```
Comment (AllentDan) [inline docs/en/benchmark.md:191]: may keep original

File: docs/en/benchmark.md [hunk 6]
```diff
@@ -303,98 +332,45 @@ Users can directly test the speed through [how_to_measure_performance_of_models.
...
-    <td align="center">COCO</td>
-    <td align="center">1x3x416x416</td>
-    <td align="center">111.60</td>
-    <td align="center">8.96</td>
-    <td align="center">134.50</td>
-    <td align="center">7.43</td>
-    <td>$MMDET_DIR/configs/yolox/yolox_tiny_8x8_300e_coco.py</td>
-  </tr>
-</tbody>
-</table>
```
Comment (AllentDan) [inline docs/en/benchmark.md:370]: deleted by mistake?

File: docs/zh_cn/benchmark.md [hunk 7]
```diff
@@ -304,98 +333,45 @@ GPU: ncnn, TensorRT, PPLNN
...
     <td align="center">-</td>
+    <td align="center">?</td>
+    <td align="center">?</td>
     <td align="center">86.80</td>
     <td align="center">11.52</td>
     <td>$MMDET_DIR/configs/mask_rcnn/mask_rcnn_r50_fpn_1x_coco.py</td>
   </tr>
 </tbody>
 </table>
-<table class="docutils">
```
Comment (AllentDan) [inline docs/zh_cn/benchmark.md:313]: deleted by mistake?

File: docs/en/benchmark.md [hunk 8]
```diff
@@ -190,13 +203,15 @@ Users can directly test the speed through [how_to_measure_performance_of_models.
     <td align="center" rowspan="3">Dataset</td>
     <td align="center" rowspan="3">Input</td>
     <td align="center" colspan="6">T4</td>
+    <td align="center" colspan="2">Jetson TX2</td>
     <td align="center" colspan="2">T4</td>
-    <td rowspan="3">model config file</td>
+    <td align="center" rowspan="3">model config file</td>
```
Comment (AllentDan) [inline docs/en/benchmark.md:208]: may remove `align="center"`

File: docs/en/benchmark.md [hunk 9]
```diff
@@ -45,17 +45,19 @@ Users can directly test the speed through [how_to_measure_performance_of_models.
     <td align="center" rowspan="3">Input</td>
     <td align="center" colspan="6">T4</td>
     <td align="center" colspan="4">JetsonNano2GB</td>
+    <td align="center" colspan="2">Jetson TX2</td>
     <td align="center" colspan="2">T4</td>
     <td align="center" colspan="2">SnapDragon888</td>
     <td align="center" colspan="2">Adreno660</td>
-    <td rowspan="3">model config file</td>
+    <td align="center" rowspan="3">model config file</td>
```
Comment (AllentDan) [inline docs/en/benchmark.md:52]: remove "center"

File: docs/en/benchmark.md [hunk 10]
```diff
@@ -324,7 +355,7 @@ Users can directly test the speed through [how_to_measure_performance_of_models.
     <td align="center" rowspan="3">Input</td>
     <td align="center" colspan="2">SnapDragon888</td>
     <td align="center" colspan="2">Adreno660</td>
-    <td rowspan="3">model config file</td>
+    <td align="center" rowspan="3">model config file</td>
```
Comment (AllentDan) [inline docs/en/benchmark.md:358]: remove center

File: docs/zh_cn/benchmark.md [hunk 11]
```diff
@@ -995,7 +1053,7 @@ GPU: ncnn, TensorRT, PPLNN
     <td align="center">fp16</td>
     <td align="center">int8</td>
     <td align="center">fp16</td>
-    <td>model config file</td>
+    <td align="center">model config file</td>
```
Comment (AllentDan) [inline docs/zh_cn/benchmark.md:1056]: remove center

File: docs/en/benchmark.md [hunk 12]
```diff
@@ -303,6 +332,8 @@ Users can directly test the speed through [how_to_measure_performance_of_models.
     <td align="center">4.14</td>
     <td align="center">-</td>
     <td align="center">-</td>
+    <td align="center">?</td>
```
Comment (lvhan028) [inline docs/en/benchmark.md:335]: What does '?' indicate?

Comment (SingleZombie) [reply docs/en/benchmark.md:335]: We have not tested this model on the platform.

**Review by AllentDan — APPROVED** [review]
LGTM

File: docs/en/benchmark.md (hunk 12 above)
Comment (RunningLeon) [reply docs/en/benchmark.md:335]: Could we use `-` as default?

**Pull request merged**